ServiceWebsite/ServiceWebsite.AcceptanceTests/TestClients/ApiTestClientBase.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using RestSharp;
+using ServiceWebsite.Common;
+
+namespace ServiceWebsite.AcceptanceTests.TestClients
+{
+    public class ApiTestClientBase : IApiClient
```

Comment:
Base class does not need to be Interfaced off

Reply:
done